.github/workflows/cdk-diff.yml

```diff
@@ -0,0 +1,26 @@
+name: CDK-DIFF
+
+on:
+  pull_request:
+    branches: [ main ]
+env:
+  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: AWS CDK GitHub Actions
+      uses: youyo/aws-cdk-github-actions@v2
+      with:
+        cdk_subcommand: 'diff'
+        actions_comment: true
+      env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: 'eu-west-1'
+      
+    - name: Run a one line example script
```

Comment:
Is this intentional or was just added for debugging and forgotten?

Reply:
I aim to remove it soon. It was for debugging.